packages/circleci/src/circleci-config.ts

```diff
@@ -146,10 +165,18 @@ export default abstract class CircleCiConfigHook {
     // properties here
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
     const jobs = (config.workflows!['tool-kit'] as Workflow).jobs!
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const nightlyJobs = (config.workflows!['nightly'] as Workflow).jobs!
     const job = this.jobOptions ? { [this.job]: this.jobOptions } : this.job
     // Avoid duplicating jobs (this can happen when check() fails when the version is wrong)
     if (!jobs.some((candidateJob) => isEqual(candidateJob, job))) {
       jobs.push(job)
+      if (this.addToNightly) {
+        // clone job and remove waiting for approvals
+        const clonedJob = JSON.parse(JSON.stringify(job))
```

Comment:
**nitpick:** this encoding/decoding can be lossy, so I usually prefer using [`lodash.clonedeep`](https://lodash.com/docs/4.17.15#cloneDeep). not a big deal in this case though